### PR TITLE
Explain why we do *not* include code examples

### LINKED ISSUE #139: Add code examples
The referee says

> More generally, given the technical nature of the work being described here, I frequently found myself wanting a small code snippet that demonstrated the new/improved functionality of the packages being described. Perhaps the authors could find a way to include short code examples where appropriate, either inline in the paper body or in the appendix.

We discussed this in the paper telecons at least once and decided that we don't like code snippets too much because (i) then can be found in the docs, (ii) they risk being version specific and could be almost out of date already when the paper is published, and (iii) they are not necessarily instructive unless you are an astropy user already and know the context well enough to fill in the missing lines (or you get very long examples).

As a compromise, I suggest that we pick no more than 5 examples with no more than 5 lines of code each. I think that these do *not* have to be self-contains and executable, e.g. say ``data=Table.read(''sourcelist.fits")`` without downloading an actual file with a persistent identifier etc.

Note that scripts for exporting code examples with syntax highlighting to pdf (to be included as a figure) exist in the repro for paper I.

### PR DESCRIPTION
**Pull request by hamogu**
I know that #139 is already assigned to somebody else, but since I wrote quite a few comments there how this could be answered, I found it easier to actually make a PR wit ha suggestion then to continue the discussion there.

You may wonder why I added to much text. I generally try to describe my motivation in detail if I directly contradict a referee's suggestion.

fixes #139

### CODE REVIEW

File: response.txt [hunk 1]
```diff
@@ -13,6 +13,43 @@ and ths statistical editor below.
...
+
+- While we aim for stability, many of the newsest features are still
+  considered provisional, meaning that their API can still change. The
+  paper describes astropy 2.0, but at the time of writing this,
+  astropy 3.0 is already released. We hope that this paper is useful
+  for years to come. Code examples for a specific version are at risk
+  of being outdated much sooner than that.
+
+- Code examples belong in the documentation. In the documentation at
+  cods.astropy.org we provide extensive examples with enough comments
```
Comment (bsipocz) [inline response.txt:33]: typo in the url

**Comment by crawfordsm** [comment]
As a compromise to the referee's comments, let's at least add links directly to the documentation for the stable version.  This will make it easier for the reader to find the code they need and will likely satisfy the requests of the referee.   

The referee would not have made the suggest to include code twice unless they thought it would significantly improve the paper, so let's save ourselves time by at least trying to compromise (instead of having to re-submit it again).   I'm happy to add the links by the end of the week if no one else has time.

**Comment by bsipocz** [comment]
@crawfordsm - Adding more links to the stable docs is a really good idea. In fact I had the misconception that we already link to there several places.

**Comment by adrn** [comment]
@crawfordsm @bsipocz Do you think we should turn all function/class names into links? Or were you thinking of adding footnotes with URLs to point to examples where relevant? Or both?

**Comment by bsipocz** [comment]
@adrn - Linking to submodules and to functions/classes is enough I think, specific examples may evolve naturally making the links broken much quicker.

**Comment by crawfordsm** [comment]
@adrn turning the functions/classes into links is a really good idea.

The one other place is providing links directly for the figures at the end of the captions to the code.   So, I think that plus linking classes and functions would satisfy this request.

**Comment by hamogu** [comment]
When you say "turn classes into a link" you mean probably mean "add a footnote with an url"? Or is there an APj compatible way to mark up LaTeX to tun text into hyperlinks?

**Pull request merged**